### PR TITLE
fix(react-icons): make sure currentColor is respected for fills

### PR DESCRIPTION
**Pull request by gdostie**
### Proposed Changes

- Instead of moving all strokes to the root of svg tags, storkes are left where they are but their value is replaced to `currentColor`. Doing this improves the quality of rendering for some icons:

Before:
image

After:
image

- Replaced all fills that were not `none` to `currentColor`

Before:
image

After:
image

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Review by GermainBergeron — APPROVED** [review]

**Comment by aiheon** [comment]
Storkes <3 :joy: 
image

File: packages/tokens/bin/lib/formats/formatSvg.ts [hunk 1]
```diff
@@ -11,16 +11,27 @@ const cleanSvgMarkup = (name: string, svgMarkup: string): string => {
...
-                params: {selector: '[stroke]', attributes: ['stroke']} as any,
-            },
             {
                 name: 'addAttributesToSVGElement',
                 active: true,
-                params: {attributes: [{stroke: 'currentColor'}, {width: '1em'}, {height: '1em'}]},
+                params: {attributes: [{width: '1em'}, {height: '1em'}]},
             },
+            {
+                type: 'visitor',
```
Comment (aiheon) [inline packages/tokens/bin/lib/formats/formatSvg.ts:20]: visitor? :thinking:

**Review by aiheon — APPROVED** [review]

File: packages/tokens/bin/lib/formats/formatSvg.ts (hunk 1 above)
Comment (gdostie) [reply packages/tokens/bin/lib/formats/formatSvg.ts:20]: its a type of svgo plugin that "visits" every html element one by one

Comment (aiheon) [reply packages/tokens/bin/lib/formats/formatSvg.ts:20]: oh I see :eyes: Interesting :+1:

**Pull request merged**